test/libsolidity/SMTChecker.cpp

```diff
@@ -38,7 +38,6 @@ class SMTCheckerFramework: public AnalysisFramework
 public:
 	SMTCheckerFramework()
```

Comment:
Do we still need the constructor in this case? At least it could be moved to the header.